crates/core/src/host/wasmtime/wasmtime_module.rs

```diff
@@ -173,11 +187,21 @@ impl module_host_actor::WasmInstancePre for WasmtimeModule {
 
         let call_procedure = get_call_procedure(&mut store, &instance);
 
+        let call_view = instance
+            .get_typed_func(&mut store, CALL_VIEW_DUNDER)
+            .expect("no call_view");
```

Comment:
How to make the CI pass until we don't have other lang module side implementation,
Should I make it optional temporarily?